rasa/nlu/selectors/response_selector.py

```diff
@@ -323,6 +326,14 @@ def preprocess_train_data(self, training_data: TrainingData) -> RasaModelData:
 
         return model_data
 
+    def _full_response(self, label: Dict[Text, Any]) -> Optional[Dict[Text, Any]]:
+        """Given a label return the full response based on the labels id."""
+        for key, responses in self.responses.items():
+            for response in responses:
+                if hash(response.get("text", "")) == label.get("id"):
```

Comment:
let's use the existing constant

Reply:
uff. the `text` of a `response` is the `RESPONSE` of a `Message`, correct? Maybe makes sense to add a comment

Reply:
yes, that is whats trained on. but as added in the other comment, this will change in the pr from daksh